roles/blacklight_app/meta/main.yml

```
@@ -1,7 +1,9 @@
 ---
 galaxy_info:
+  role_name: blacklight_app
   author: Princeton University Library
```

This should fix the current test failure:
```suggestion
  company: Princeton University Library
```